great_ape_safe/great_ape_safe.py

```diff
@@ -252,6 +255,71 @@ def contract(self, address, Interface=None, from_explorer=False):
         else:
             return Contract(address, owner=self.account)
 
+    def post_safe_tx_manually(self):
+        safe_tx = self.post_safe_tx(events=False, silent=True, post=False)
```

Comment:
in the case that there is already a partial signature, wouldn't it better to get the `safe_tx` from `_get_safe_tx_by_nonce`?. what happens if signers run the same script but produce different calldata?

Reply:
that func relies on the gnosis api which the tx is never posted to---the whole idea here is to avoid that web2 api

your point is valid though; if the locally run tx differs slightly (ie diff min out calc based on market or something) the calldata will be diff.

maybe final calldata should be passed from signers as well, just like the partial sig